src/components/Message/MessageCommerce.js

```diff
@@ -200,7 +218,78 @@ class MessageCommerce extends PureComponent {
         </div>
       </div>
     );
-  }
+  };
+
+  renderMessageText = () => {
+    const {
+      actionsEnabled,
+      handleReaction,
+      message,
+      onMentionsClickMessage,
+      onMentionsHoverMessage,
+      unsafeHTML,
+      t,
+    } = this.props;
+    const hasAttachment = this.hasAttachments();
+    const hasReactions = this.hasReactions();
+
+    if (!message) {
+      return null;
+    }
+
+    return (
+      <div className="str-chat__message-commerce-text">
+        <div
+          data-testid="message-commerce-text-inner-wrapper"
+          className={`str-chat__message-commerce-text-inner
+									${hasAttachment ? 'str-chat__message-commerce-text-inner--has-attachment' : ''}
```

Comment:
indentation ... something looks wrong

Reply:
Should be fixed here: https://github.com/GetStream/stream-chat-react/pull/283/commits/8ebde1ebd291c811cbcd57afaf05fd4a91a762db#diff-f031ec7f40f462e71c8f6641a01aa62fR245